metadata-ingestion/src/datahub/ingestion/source/sql/mssql/job_models.py

```diff
@@ -11,18 +15,65 @@
 
 @dataclass
 class ProcedureDependency:
+    flow_id: str
+    env: str
+    server: Optional[str]
     db: str
     schema: str
     name: str
     type: str
-    env: str
-    server: Optional[str]
+    incoming: Union[str, int]
+    outgoing: Union[str, int]
     source: str = "mssql"
 
+    def __post_init__(self):
+        self.incoming = int(self.incoming)
+        self.outgoing = int(self.outgoing)
```

Comment:
can we convert things into an int before passing into this model? if `incoming` is always an int, I want the type annotation to match

Reply:
Yes. Fixed.